tracker/utils.py

```diff
@@ -260,3 +260,24 @@ def get_user_revisions(telegram_id: str) -> list[dict]:
                 return_data["reviews"] = reviews_data
                 reviews_list.append(return_data.copy())
     return reviews_list
+
+def get_contributor_issues(username: str) -> list:
+    """
+    Retrieves all issues for assigned to the user github.
```

Comment:
Rework this function description to be more meaningful

Reply:
Could you give me a hand with this please?. The issue description isn´t very meaningful either

Reply:
> Could you give me a hand with this please?. The issue description isn´t very meaningful either

I meant the English part of your docstring. Let it be "Retrieves all issues the user is assigned to" for example